services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -141,6 +142,8 @@ protected void doControllerMsg(CommandMessage message) {
 
     private void handleCommand(CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
+        logger.info("Handling message: '{}'. Topic: '{}'. Destination: '{}'.",
+                message, replyToTopic, replyDestination);
```

Comment:
!!!IMPORTANT COMMENT!!!

Here we log message data, topic and destination.

This data is used bellow in do***() methods so there is no need to log all this data it do***() methods